app/src/main/java/br/org/cesar/discordtime/stickysessions/ui/session/SessionActivity.java

```diff
@@ -270,14 +270,14 @@ public void displayError(String message) {
     }
 
     @Override
-    public void startLoadingSession() {
-        mLoadingView.setVisibility(View.VISIBLE);
+    public void startLoading() {
+        mLoading.show();
```

Comment:
Please check visually which approach is better. The previous loading has a good looking but we would need to implement a specific loading for the note on contract.